web/styles/app_variables.css

```diff
@@ -1636,6 +1835,211 @@
         226.35deg 82.53% 55.1% / 38.82%
     );
     --color-typeahead-option-label: var(--grey-400);
+
+    /* Buttons -- Neutral Variant */
+    --color-neutral-primary-btn-text: color-mix(
```

Comment:
Is there a reason to use `btn` rather than `button` in these names? We're trying to purge that abbreviation.

Reply:
Do we want these variables to use `-action-button-` or just `-button-`?